packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -120,6 +120,68 @@ lambdaAction.outputArtifact('Out2'); // returns the named output Artifact, or th
 See [the AWS documentation](https://docs.aws.amazon.com/codepipeline/latest/userguide/actions-invoke-lambda-function.html)
 on how to write a Lambda function invoked from CodePipeline.
 
+### Lambda deployed through CodePipeline
+
+If you want to deploy your Lambda through CodePipeline,
+you can use the special `Code` convenience method, `lambda.Code.codePipeline`.
+Note that your Lambda must be in a separate Stack than your Pipeline.
+The Lambda itself will be deployed, alongside the entire Stack it belongs to,
+using a CloudFormation CodePipeline Action. Example:
+
+```typescript
+const lambdaCode = lambda.Code.codePipeline(); 
```

Comment:
Feels like it magically appears? I would have expected to provide some reference to the pipeline.

Reply:
Changed the name to `cfnParameters()`.